spec/src/main/asciidoc/microprofile-opentracing.asciidoc

```diff
@@ -190,10 +190,10 @@ When the `@Traced(false)` annotation is used for a JAX-RS endpoint method, the u
 * `operationName=<Name for the Span>`.
 Default is `""`.
 If the `@Traced` annotation finds the `operationName` as `""`, the default operation name is used. For a JAX-RS endpoint method (see <<server-span-name>>).
-If the annotated method is not a JAX-RS endpoint, the default operation name of the new Span for the method is
-```
-<package name>.<class name>.<method name>
-```
+If the annotated method is not a JAX-RS endpoint, the default operation name of the new Span for the method is `<package name>.<class name>.<method name>`.
```

Comment:
In my opinion, this whole logic looks a bit too fancy and harder to understand for end users. I am against this change.

If users want to override the name then we should give them a way to actually do it.  Imagine you trace two different impls but with the same op name, this can be used later on in aggregation or analytics job.  

Spans usually contain other tags which can be used for query purposes (e.g. http tags) there is no need to encode everything in the operation name.

Reply:
Updated in latest commit